jobs/grafana/templates/post-start

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
 
-echo "[$(date)] Calling 'create-update-datasources' ..."
-$(dirname $0)/create-update-datasources
-
 echo "[$(date)] Calling 'create-update-dashboards' ..."
 $(dirname $0)/create-update-dashboards
```

Comment:
Is this still necessary?

Reply:
I think it is because something needs to write the dashboard json files to disk so the provisioner can pick them up.

If it was a single dashboard json then the BOSH templating could create the file like I'm doing with datasources.yml. But in this case it can be a list of dashboards so multiple files may need to be written.

Reply:
I can't find any option in grafana to support having multiple dashboards in a single json file. Closest I've found is [this thread](https://community.grafana.com/t/how-to-create-multiple-dashboard-using-dashboard-provisioning/6429) which never got a real response.